x/auth/types/types.proto

```diff
@@ -0,0 +1,66 @@
+syntax = "proto3";
+package cosmos_sdk.x.auth.v1;
+
+import "third_party/proto/gogoproto/gogo.proto";
+import "types/types.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/x/auth/types";
+
+// Account defines the default concrete account type used in the x/auth module.
+// It contains a single oneof field member, BaseAccountProto. The type implements
+// the Account interface and provides all necessary account functionality.
+//
+// This type should only be used internally for testing purposes. The application's
+// concrete Account type should be defined in the application-level codec.
+//
+// Applications may extend account functionality and types by extending the
+// oneof set and using their own AuthCodec (e.g. types/vesting).
+message Account {
```

Comment:
Maybe call this `BasicAccount` or `SimpleAccount` instead?

Reply:
We don't even need this type -- we have the app codec that has everything we need. I removed this type and all boilerplate/cruft code.